### PR TITLE
chore: cleanup firebase libraries

### LINKED ISSUE #120: Cleanup firebase libraries
Remove `@nuxtjs/firebase` and `@vueuse/firebase`

### PR DESCRIPTION
**Pull request by razbakov**
Closes #120

### CODE REVIEW

File: components/TInput/TInputCity.vue [hunk 1]
```diff
@@ -78,10 +78,11 @@ import { computed } from '@nuxtjs/composition-api'
 import { getLocation, sanitize, sortBy } from '~/utils'
 import { useDoc } from '~/use/doc'
 import { useCollection } from '~/use/collection'
+import { googleApiKey } from '~/plugins/firebase'
 
 Vue.use(VueGoogleMaps, {
   load: {
-    key: process.env.firebase.config.apiKey,
+    key: googleApiKey,
```
Comment (best-lucky1030) [inline components/TInput/TInputCity.vue:85]: `googleApiKey` is from `plugins/firebase`?
I mean the value of `googleApiKey`.

File: plugins/firebase.js [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
+let googleApiKey
+
+if (!firebase.apps.length) {
+  const configString = process.env.FIREBASE_CONFIG
+  if (!configString) {
+    throw new Error('FIREBASE_CONFIG is not defined')
+  }
+
+  const config = JSON.parse(configString)
+  googleApiKey = config.apiKey
```
Comment (best-lucky1030) [inline plugins/firebase.js:23]: seeing this, `googleApiKey` is from config env.

File: components/TInput/TInputCity.vue (hunk 1 above)
Comment (razbakov) [reply components/TInput/TInputCity.vue:85]: Yes

File: plugins/firebase.js (hunk 2 above)
Comment (razbakov) [reply plugins/firebase.js:23]: Exactly

**Pull request merged**